api/Navigator.json

```diff
@@ -837,10 +837,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/getBattery",
           "support": {
             "chrome": {
-              "version_added": "45"
+              "version_added": "38"
```

Comment:
Confirmed that the Opera data matches this. (Not changed in this PR because it was already correct.)